frontend/src/js/components/settings/PlanExpanded.tsx

```diff
@@ -14,83 +14,115 @@
 import { useRef, useState } from 'react';
 import { useSelector } from 'react-redux';
 
-import { Divider, Drawer } from '@mui/material';
+import { Button, Divider, Drawer } from '@mui/material';
+import { makeStyles } from 'tss-react/mui';
 
 import { DrawerTitle } from '@northern.tech/common-ui/DrawerTitle';
 import Form from '@northern.tech/common-ui/forms/form';
 import storeActions from '@northern.tech/store/actions';
 import { Plan } from '@northern.tech/store/constants';
-import { Organization } from '@northern.tech/store/organizationSlice/types';
+import { BillingProfile, Card, Organization } from '@northern.tech/store/organizationSlice/types';
 import { getCurrentUser } from '@northern.tech/store/selectors';
 import { useAppDispatch } from '@northern.tech/store/store';
-import { completeUpgrade } from '@northern.tech/store/thunks';
+import { completeUpgrade, editBillingProfile } from '@northern.tech/store/thunks';
 
+import CardSection from './CardSection';
 import { PlanExpandedForm } from './PlanExpandedForm';
-import CardSection from './cardsection';
+import OrganizationPaymentSettings from './organization/OrganizationPaymentSettings';
 
 const { setSnackbar } = storeActions;
 
-interface PlanExpandedProps {
-  plan: Plan;
+interface PlanExpandedPropsBase {
   organization: Organization;
   onCloseClick: () => void;
+  isEdit: boolean;
+}
+interface ProfileEditProps extends PlanExpandedPropsBase {
+  isEdit: true;
+  currentBillingProfile: BillingProfile;
+  card: Card;
+}
+
+interface PlanProps extends PlanExpandedPropsBase {
+  isEdit: false;
+  plan: Plan;
 }
+
+const useStyles = makeStyles()(() => ({
+  submitButtonContainer: {
+    position: 'relative',
+    top: '172px',
+    height: 0
```

Comment:
I assume this is to position the buttons below the "Payment card" section while the payment details are outside of the `Form` - this will break easily + the positioning is probably not going to play nice with tests (`height: 0` might not be considered accessible for an element e.g.) - please adjust this to be part of the form or handle the form actions outside of the form